from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_SetPutar(object):
    def setupUi(self, SetPutar):
        SetPutar.setObjectName("SetPutar")
        SetPutar.resize(1024, 600)
        SetPutar.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.SETWAKTU = QtWidgets.QGroupBox(SetPutar)
        self.SETWAKTU.setGeometry(QtCore.QRect(0, 0, 512, 600))
        self.SETWAKTU.setStyleSheet("background-color: transparent;\n"
"border-style: none;")
        self.SETWAKTU.setTitle("")
        self.SETWAKTU.setObjectName("SETWAKTU")
        self.SETJAM_2 = QtWidgets.QSpinBox(self.SETWAKTU)
        self.SETJAM_2.setGeometry(QtCore.QRect(122, 87, 267, 150))
        font = QtGui.QFont()
        font.setPointSize(50)
        font.setBold(True)
        font.setWeight(75)
        self.SETJAM_2.setFont(font)
        self.SETJAM_2.setStyleSheet("")
        self.SETJAM_2.setMaximum(24)
        self.SETJAM_2.setDisplayIntegerBase(10)
        self.SETJAM_2.setObjectName("SETJAM_2")
        self.SETJAM = QtWidgets.QLabel(self.SETWAKTU)
        self.SETJAM.setGeometry(QtCore.QRect(152, 12, 208, 75))
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(35)
        font.setBold(True)
        font.setWeight(75)
        self.SETJAM.setFont(font)
        self.SETJAM.setStyleSheet("background-color: transparent;")
        self.SETJAM.setTextFormat(QtCore.Qt.PlainText)
        self.SETJAM.setScaledContents(False)
        self.SETJAM.setAlignment(QtCore.Qt.AlignCenter)
        self.SETJAM.setObjectName("SETJAM")
        self.SETWAKTU_3 = QtWidgets.QSpinBox(self.SETWAKTU)
        self.SETWAKTU_3.setGeometry(QtCore.QRect(122, 312, 267, 150))
        font = QtGui.QFont()
        font.setPointSize(50)
        font.setBold(True)
        font.setWeight(75)
        self.SETWAKTU_3.setFont(font)
        self.SETWAKTU_3.setStyleSheet("")
        self.SETWAKTU_3.setMaximum(60)
        self.SETWAKTU_3.setDisplayIntegerBase(10)
        self.SETWAKTU_3.setObjectName("SETWAKTU_3")
        self.SETWAKTU_4 = QtWidgets.QLabel(self.SETWAKTU)
        self.SETWAKTU_4.setGeometry(QtCore.QRect(131, 237, 249, 75))
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(32)
        font.setBold(True)
        font.setWeight(75)
        self.SETWAKTU_4.setFont(font)
        self.SETWAKTU_4.setStyleSheet("background-color: transparent;")
        self.SETWAKTU_4.setTextFormat(QtCore.Qt.PlainText)
        self.SETWAKTU_4.setScaledContents(False)
        self.SETWAKTU_4.setAlignment(QtCore.Qt.AlignCenter)
        self.SETWAKTU_4.setObjectName("SETWAKTU_4")
        self.BACK_2 = QtWidgets.QPushButton(self.SETWAKTU)
        self.BACK_2.setGeometry(QtCore.QRect(68, 479, 375, 70))
        self.BACK_2.setText("")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("media/Frame 4.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.BACK_2.setIcon(icon)
        self.BACK_2.setIconSize(QtCore.QSize(375, 70))
        self.BACK_2.setObjectName("BACK_2")
        self.DisplaySuhu = QtWidgets.QGroupBox(SetPutar)
        self.DisplaySuhu.setGeometry(QtCore.QRect(512, 0, 512, 600))
        self.DisplaySuhu.setStyleSheet("background-color: transparent;\n"
"border-style: none;")
        self.DisplaySuhu.setTitle("")
        self.DisplaySuhu.setObjectName("DisplaySuhu")
        self.TERMO = QtWidgets.QLabel(self.DisplaySuhu)
        self.TERMO.setGeometry(QtCore.QRect(62, 170, 100, 300))
        self.TERMO.setText("")
        self.TERMO.setPixmap(QtGui.QPixmap("media/TERMO.png"))
        self.TERMO.setScaledContents(True)
        self.TERMO.setObjectName("TERMO")
        self.BACK = QtWidgets.QPushButton(self.DisplaySuhu)
        self.BACK.setGeometry(QtCore.QRect(64, 479, 375, 70))
        self.BACK.setText("")
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("media/Frame 3.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.BACK.setIcon(icon1)
        self.BACK.setIconSize(QtCore.QSize(375, 70))
        self.BACK.setObjectName("BACK")
        self.JUDUL = QtWidgets.QLabel(self.DisplaySuhu)
        self.JUDUL.setGeometry(QtCore.QRect(20, 11, 464, 75))
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(35)
        font.setBold(True)
        font.setWeight(75)
        self.JUDUL.setFont(font)
        self.JUDUL.setStyleSheet("background-color: transparent;")
        self.JUDUL.setText("")
        self.JUDUL.setTextFormat(QtCore.Qt.PlainText)
        self.JUDUL.setPixmap(QtGui.QPixmap("media/JUDUL SET.png"))
        self.JUDUL.setScaledContents(True)
        self.JUDUL.setAlignment(QtCore.Qt.AlignCenter)
        self.JUDUL.setObjectName("JUDUL")
        self.SUHU = QtWidgets.QLabel(self.DisplaySuhu)
        self.SUHU.setGeometry(QtCore.QRect(186, 215, 230, 234))
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(140)
        font.setBold(True)
        font.setWeight(75)
        self.SUHU.setFont(font)
        self.SUHU.setStyleSheet("background-color: transparent;")
        self.SUHU.setTextFormat(QtCore.Qt.PlainText)
        self.SUHU.setAlignment(QtCore.Qt.AlignCenter)
        self.SUHU.setObjectName("SUHU")
        self.DERAJAD = QtWidgets.QLabel(self.DisplaySuhu)
        self.DERAJAD.setGeometry(QtCore.QRect(400, 190, 40, 82))
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(50)
        font.setBold(True)
        font.setWeight(75)
        self.DERAJAD.setFont(font)
        self.DERAJAD.setStyleSheet("background-color: transparent;")
        self.DERAJAD.setTextFormat(QtCore.Qt.PlainText)
        self.DERAJAD.setAlignment(QtCore.Qt.AlignCenter)
        self.DERAJAD.setObjectName("DERAJAD")
        self.Logo = QtWidgets.QLabel(SetPutar)
        self.Logo.setGeometry(QtCore.QRect(514, 21, 535, 507))
        self.Logo.setStyleSheet("background-color: transparent;")
        self.Logo.setText("")
        self.Logo.setPixmap(QtGui.QPixmap("media/TELKOM2.png"))
        self.Logo.setScaledContents(True)
        self.Logo.setObjectName("Logo")
        self.Logo.raise_()
        self.SETWAKTU.raise_()
        self.DisplaySuhu.raise_()

        self.retranslateUi(SetPutar)
        QtCore.QMetaObject.connectSlotsByName(SetPutar)
    

    def retranslateUi(self, SetPutar):
        _translate = QtCore.QCoreApplication.translate
        SetPutar.setWindowTitle(_translate("SetPutar", "Form"))
        self.SETJAM.setText(_translate("SetPutar", "SET JAM"))
        self.SETWAKTU_4.setText(_translate("SetPutar", "SET MENIT"))
        self.SUHU.setText(_translate("SetPutar", "30"))
        self.DERAJAD.setText(_translate("SetPutar", "o"))
